Error: Cannot find module '#pino/file'

## Brief Description

I set up the minimum environment to run this example:

```
const pino = require('pino');
const transport = pino.transport({
        target: '#pino/file',
});
pino(transport);
```

But it turns out to show `Error: Cannot find module '#pino/file'`.

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '#pino/file'
Require stack:
- /private/tmp/test/main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19)
    at fixTarget (/private/tmp/test/node_modules/pino/lib/transport.js:102:30)
    at Function.transport (/private/tmp/test/node_modules/pino/lib/transport.js:87:22)
    at Object.<anonymous> (/private/tmp/test/main.js:2:24)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/private/tmp/test/main.js' ]
}
```

## Reproducible Step

I'm in `darwin/arm64`.

```
$ node -v
v16.11.1
$ yarn add pino && cat << EOF >> main.js
const pino = require('pino');
const transport = pino.transport({
        target: '#pino/file',
});
pino(transport);
EOF
$ node main.js
```

--- a/docs/transports.md
+++ b/docs/transports.md
@@ -212,7 +212,7 @@ The `options.destination` property may be set to specify the desired file destin
 ```js
 const pino = require('pino')
 const transport = pino.transport({
-  target: '#pino/file',
+  target: 'pino/file',
   options: { destination: '/path/to/file' }
 })
 pino(transport)
@@ -220,7 +220,7 @@ pino(transport)
 
 The `options.destination` property may also be a number to represent a file descriptor. Typically this would be `1` to write to STDOUT or `2` to write to STDERR. If `options.destination` is not set, it defaults to `1` which means logs will be written to STDOUT.
 
-The difference between using the `#pino/file` transport builtin and using `pino.destination` is that `pino.destination` runs in the main thread, whereas `#pino/file` sets up `pino.destination` in a worker thread.
+The difference between using the `pino/file` transport builtin and using `pino.destination` is that `pino.destination` runs in the main thread, whereas `pino/file` sets up `pino.destination` in a worker thread.
 
 #### `pino-pretty`
 

